cypress/integration/special-pages-spec.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="Cypress" />
+
+import { ArticlePage } from '../page-objects/article-page'
+
+const articlePage = new ArticlePage()
+
+describe('Special Pages', () => {
+  beforeEach(() => {
+    cy.navigateToHomePage()
+  })
+
+  it('special page article should open', () => {
+    cy.visit('http://127.0.0.1:8080/#/article/en/Help%3AIPA%2FEnglish')
+    articlePage.title().should('have.text', 'Help:IPA/English')
+    cy.get('.article-content').should('contain.text', 'Throughout Wikipedia, the pronunciation of words is indicated by means of the ')
+  })
+})
```

Comment:
not sure how special it is? are you going to test that opening the preview message, then goes to this article page?

Reply:
right now I can't select the link I want to open on the preview , will complement the test when I can